test.properties

```diff
@@ -7721,6 +7721,27 @@ test.batch.run.property.query[functional-tomcat90-mysql57-jdk8][echo-db-partitio
             (testray.main.component.name ~ "Upgrades Staging")\
         )
 
+    #
+    # Staging with Custom Database
+    #
+
+    custom.database.file.name=LRQA-78789-mysql
```

Comment:
As we don't upload this db dump, we might want this property commented out at the moment, and replaced with empty value